### ISSUE TITLE
Version 4.3.0 Bootstrap fail and block after disconnection

### ISSUE THREAD

**Issue by Andrea4Nav** [NAVCoin/navcoin-core#254]
I just have few suggestion, the link for the bootsrap should be incorporated and autoupdated (will be very nice) so no need to insert it manually, than i try to disconnect during the bootstrap and when i do it again it start from the beginning and after a few block it stop, so i have to delete the old file in order to start the download again

**Comment by aguycalled** [comment]
could you elaborate how to replicate this issue?

**Comment by Andrea4Nav** [comment]
i started the bootsrap from the wallet, than interrupt the internet connection while downloading, when connecting again to internet it not continue/restart by itself i have to close the wallet (forced), i restart with a new one and fail, than i deleted the old file and starting again with good result.

**Comment by aguycalled** [comment]
you don't really need to remove the old file, the client takes care of this. is the issue you are describing basically that the download does not resume after the connection is restablished?

**Comment by Andrea4Nav** [comment]
yes, also it stuck when i try to do it again, (maybe was just my connection problem) so i delete the old file and try again, doing this it worked.
